src/VisualStudio/Core/Impl/CodeModel/ProjectCodeModelFactory.cs

```diff
@@ -32,15 +38,91 @@ internal sealed class ProjectCodeModelFactory : IProjectCodeModelFactory, IDispo
         /// </summary>
         private readonly JoinableTaskCollection _deferredCleanupTasks;
 
+        /// <summary>
+        /// Cancellation token that controls existing async work that we have kicked off.  Canceled when we're finally
+        /// disposed.
+        /// </summary>
+        private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
+
+        private readonly IForegroundNotificationService _notificationService;
+        private readonly IAsynchronousOperationListener _listener;
+        private readonly AsyncBatchingWorkQueue<DocumentId> _documentsToFireEventsFor;
+
         [ImportingConstructor]
         [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
```

Comment:
📝 If this is no longer used in tests, this attribute should be removed and replaced with the `Obsolete` attribute form.

Reply:
No.  I just didn't actually add/submit my test chnages :-o